DOCS-942: Add MoveOnGlobe() to the Motion Service docs (#1570)
viamrobotics/docs: docs/program/apis/_index.md

[`Move`](/services/motion/#move) | Move multiple components in a coordinated way to achieve a desired motion.
[`MoveSingleComponent`](/services/motion/#movesinglecomponent) | Move a single component "manually."
[`GetPose`](/services/motion/#getpose) | Get the current location and orientation of a component.

> [`MoveOnMap`](/services/motion/#moveonmap) | Move a component to a `Pose` in respect to the origin of a [SLAM](/services/slam/) map.

Author: ```suggestion
[`MoveOnMap`](/services/motion/#moveonmap) | Move a [base](/component/base/) component to a `Pose` in respect to the origin of a [SLAM](/services/slam/) map.
```

> [`MoveOnMap`](/services/motion/#moveonmap) | Move a [base](/component/base/) component to a `Pose` in respect to the origin of a [SLAM](/services/slam/) map.

Author: ```suggestion
[`MoveOnMap`](/services/motion/#moveonmap) | Move a [base](/components/base/) component to a `Pose` in respect to the origin of a [SLAM](/services/slam/) map.
```

> [`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a component to a specific latitude and longitude, using a [Movement Sensor](/components/movement-sensor/) to determine the location.

Author: ```suggestion
[`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.
```

> [`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.

Author: @npentrel I was originally thinking `Move a component to a specific latitude and longitude, using a [Movement Sensor](/components/movement-sensor/) to determine the location.` for this line but i think that could be a bit overwhelming. Thought this was easier to parse, suggestions welcome

Reviewer: sounds good to me.

> [`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.

Reviewer 2: Should add that MoveOnGlobe (and MoveOnMap) is only supported for bases.

Author: Done

> [`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.

Author: ```suggestion
[`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a [base](/component/base/) component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.
```

> [`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a [base](/component/base/) component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.

Author: ```suggestion
[`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a [base](/components/base/) component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.
```
